README.md

```diff
@@ -204,6 +204,41 @@ service.metadata.labels.role = 'slave'
 client.create_service service`
 ```
 
+Below example is for the extension API
```

Comment:
Added some more mentions. A lot of the places seem like they just list examples of what you can do, and don't necessarily need to mention all possible operations.